README.md

```diff
@@ -618,9 +234,17 @@ npm start
 # Observe services tracing information with Jaeger
 
 After running the docker container, you can view the tracing information on Jaeger via http://localhost:16686/. You can select the service in the drop-down box as follows.
-![image info](drop-down-services-spans.png)
+![image info](images/drop-down-services-spans.png)
 
 Then click on `Find Traces` and you will be able to view spans of services in the gcp-microservice.
-![image info](tracing-spans.png)
+![image info](images/tracing-spans.png)
 
 For more information about observability in Ballerina, please visit [Observe Ballerina Programs](https://ballerina.io/learn/observe-ballerina-programs/).
+
+# Deviations from [GCP Microservices Demo](https://github.com/GoogleCloudPlatform/microservices-demo)
```

Comment:
We need to mention that we broke down frontend service to two services. UI and frontned(BFF service we agreed that day)

Reply:
Added.